doc/user_guide/Running_in_Webassembly.md

```diff
@@ -0,0 +1,176 @@
+# Running in Webassembly
+
+Panel lets you write dashboards and other applications in Python that are accessed using a web browser. Typically, the Python interpreter runs as a separate Jupyter or Bokeh server process, communicating with JavaScript code running in the client browser. However, it is now (as of 2022) possible to run Python directly in the browser, with no separate server needed!
```

Comment:
remove (as of 2022). Its not necessary and in 1 or 2 years someone will see the documentation as outdated due to this.